src/App.test.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import App from './App';
 
+jest.mock('./components/common/SwipeComponent/SwipeComponent', () => 'SwipeComponent');
```

Comment:
Wszystkie te wywołania usuń proszę, ta instrukcja nic nam nie daje, a jeżeli wynika to z generowanego błędu, to popracuj nad tym dlaczego ten bląd leci.

Reply:
jasne chodzi pewnie o brak wsparcia dla Swipe'a